tests/lib/nm/wired_test.py

```diff
@@ -49,10 +49,19 @@ def test_create_setting_mtu(NM_mock):
 
 
 def test_create_setting_auto_negotiation_False(NM_mock):
-    setting = nm.wired.create_setting(
-        {'ethernet': {'auto-negotiation': False}}, None)
+    ethtool_result = {'speed': 1337, 'duplex': 'mocked', 'auto-negotiation':
+                      'mocked'}
+
+    with mock.patch('libnmstate.nm.wired.minimal_ethtool',
```

Comment:
nit: You could have used a decorator for this:
```
@mock.patch.object(nm.wired, 'minimal_ethtool', return_value={'speed': 1337, 'duplex': 'mocked', 'auto-negotiation': 'mocked'})
def test_create_setting_auto_negotiation_False(NM_mock, mock_ethtool):
```
(I may be wrong about the order of the args in the test function)

I'm ok with the current version as well, let me know if I should wait or accept as is.

Reply:
if made it a decorator